content/_includes/enterprise-only.md

```diff
@@ -0,0 +1,2 @@
+This feature is available for all [Enterprise](https://wandb.ai/site/pricing/) licenses.
```

Comment:
Should this be "only available?"

Reply:
Yeah it needs to be a little more detailed, getting the facts straight now. Thanks!